src/v/kafka/server/partition_proxy.cc

```diff
@@ -18,13 +18,25 @@
 namespace kafka {
 
 std::optional<partition_proxy> make_partition_proxy(
-  const model::ntp&,
-  ss::lw_shared_ptr<cluster::partition> partition,
-  cluster::partition_manager&) {
-    /// Render materialized_partition unused in this commit only. In a
-    /// subsequent commit a new method will be used to decide weather a
-    /// partition_proxy is materialized or not
-    return make_partition_proxy<replicated_partition>(partition);
+  const model::ntp& ntp,
+  cluster::metadata_cache& md_cache,
+  cluster::partition_manager& pm) {
```

Comment:
This will break repository bisectabilty since this commit would'n comile since call site isn't updated

Reply:
Ensured all commits build with next force-push